core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/provider.go

```diff
@@ -80,7 +80,8 @@ var _ LogEventProviderTest = &logEventProvider{}
 type logEventProvider struct {
 	lggr logger.Logger
 
-	cancel context.CancelFunc
+	cancel  context.CancelFunc
+	threads sync.WaitGroup
```

Comment:
/nit since this a WaitGroup, can we make the naming of this more explicit/match with the suffix `-WG` naming pattern we use in a few other repos?

`threadsWG`